src/visualizers/widgets/ArtifactIndex/ModalControl.js

```diff
@@ -0,0 +1,31 @@
+/*globals define, $*/
+define([
+    'deepforge/viz/Utils',
+    'text!./ArtifactModal.html'
+], function(
+    Utils,
+    MODAL_HTML
+) {
+    'use strict';
+
+    const ModalControl = function() {
+        this.$modal = $(MODAL_HTML);
+        this.$modalTitle = this.$modal.find('#artifactName');
```

Comment:
It would be better (and consistent with the following lines) to look up using a class name here. As a minor style point, it might be good to try to follow a single convention and use `.artifact-name`.